frontend/pages/sites/SiteListItem.test.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+
+import SiteListItem from './SiteListItem';
+// import GitHubLink from '@shared/GitHubLink';
+// import PublishedState from './PublishedState';
+
+const mockOrganization = {
+  name: 'Test Organization',
+  isActive: true,
+  isSandbox: false,
+  daysUntilSandboxCleaning: 0,
+  id: '1234',
+};
+
+const mockSite = {
+  repository: 'test-repo',
+  owner: 'test-owner',
+  id: 1,
+  publishedAt: '2024-12-18T00:00:00Z',
+  repoLastVerified: '2024-12-17T00:00:00Z',
+  createdAt: '2024-12-01T00:00:00Z',
+  viewLink: '/sites/1/builds',
+  isActive: true,
+};
+
+describe('<SiteListItem />', () => {
+  it('renders the organization name in a heading element', () => {
+    render(
+      /* MemoryRouter allows us to render components that use react Link  */
+      <MemoryRouter>
```

Comment:
This seems like something we might wanna boilerplate out? 

```import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

export const renderWithRouter = (ui, { route = '/' } = {}) => {
  return render(
    <MemoryRouter initialEntries={[route]}>
      {ui}
    </MemoryRouter>
  );
};```

Reply:
It seems simple enough for now to leave it. Breaking out the a component doesn't simplify or clean up the tests.